app/src/main/java/fr/free/nrw/commons/depictions/WikidataItemDetailsActivity.java

```diff
@@ -0,0 +1,204 @@
+package fr.free.nrw.commons.depictions;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.FrameLayout;
+
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.viewpager.widget.ViewPager;
+
+import com.google.android.material.tabs.TabLayout;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import fr.free.nrw.commons.Media;
+import fr.free.nrw.commons.R;
+import fr.free.nrw.commons.depictions.Media.DepictedImagesFragment;
+import fr.free.nrw.commons.depictions.SubClass.SubDepictionListFragment;
+import fr.free.nrw.commons.explore.ViewPagerAdapter;
+import fr.free.nrw.commons.media.MediaDetailPagerFragment;
+import fr.free.nrw.commons.theme.NavigationBaseActivity;
+import fr.free.nrw.commons.upload.structure.depictions.DepictedItem;
+
+/**
+ * Activity to show depiction media, parent classes and child classes of depicted items in Explore
+ */
+public class WikidataItemDetailsActivity extends NavigationBaseActivity implements MediaDetailPagerFragment.MediaDetailProvider, AdapterView.OnItemClickListener {
```

Comment:
Use MVP pattern for this activity.

Reply:
I am happy to make minor corrections but this seems too large for the intention of getting this on master to stop further divergence and so it can be improved. I would make a separate ticket.
Pinging @nicolas-raoul for input

Reply:
For this particular branch I am in favor of merging first and fixing later.
Breaking our long-time rule is for @misaochan to allow/disallow, though.